### ISSUE TITLE
Create Wi-Fi setup onboarding page

### ISSUE THREAD

**Issue by rock3r** [squanchy-dev/squanchy-flutter#46]
The second onboarding page is the WiFi setup page; it should look something along these lines:

![image](https://user-images.githubusercontent.com/153802/32701856-38a8716a-c7dd-11e7-9f8d-724f0eebc1f3.png)

Please refer to @ScribblyPixels' designs.

The "no thanks" button brings users to the next page, "yes please" starts the automatic setup process. If setup succeeds they're immediately brought to the next onboarding page (or schedule if this is the last one), otherwise they remain on the current page and some error is displayed (out of scope, see #todo).

⚠️ **The existing designs are still WIP**

⚠️ Blocked by #48 (implementing the WiFi setup code)

🗒 Wi-Fi setup is only available on [iOS 11+](https://developer.apple.com/documentation/networkextension/nehotspotconfiguration)

🗒 This page can only be shown if the app has the WiFi configuration data, otherwise it must be skipped.
